app/assets/javascripts/application.js

```diff
@@ -378,6 +407,12 @@ $(document).ready(function() {
   })
 });
 
+// Sync Handling
+$(document).ready(function() {
```

Comment:
Does putting this in `$(document).ready` play nice with turbolinks? If you click a filter whilst it's syncing, does it continue to check?

Reply:
seems to work!

Reply:
👍